ocs_ci/utility/utils.py

```diff
@@ -4739,3 +4739,21 @@ def exec_nb_db_query(query):
         output = output[2:-1]
 
     return output
+
+
+def get_latest_release_version():
```

Comment:
This function is in generic utils module. Please add information about what latest release version is obtained by the function into function name and description.

Reply:
added, thanks